SimG4CMS/Forward/interface/PltSD.h

```diff
@@ -49,32 +44,29 @@ class PltSD : public SensitiveTkDetector,
 private:
 
   virtual void   sendHit();
-  virtual void   updateHit(G4Step *);
-  virtual bool   newHit(G4Step *);
-  virtual bool   closeHit(G4Step *);
-  virtual void   createHit(G4Step *);
+  virtual void   updateHit(const G4Step *);
```

Comment:
why virtual?
PltSD does not seem to have children. Most probably can be made ```final```

Reply:
Will fix in the next iteration.